doc/sphinx/using/basics/access_management.rst

```diff
@@ -0,0 +1,86 @@
+
+Centralized (ask the IT crowd)
+******************************
+
+Another way to handle this can be to grant a limited number of people, such as security teams, the right to add personal accesses to others. This way people are less autonomous, but it might be useful if adding accesses has to be enacted via normalized processes. It also has some nice effects: as a sysadmin, one of the pros is that you can create 3 separate accounts on the remote machine, and map them to each bastion account you're adding. This is a good method for achieving **end-to-end traceability**; including on the remote server; where you might want to install **auditd** or similar tools. It's also doable in the help yourself mode, but it may be harder to enforce.
+
+To be clear, this access model doesn't scale so efficiently when we're dealing with whole teams, or big infrastructures – this is where group-based access comes handy.
+
+.. _accessManagementGroupAccesses:
+
+Group Accesses
+==============
+
+.. image:: /img/groups.png
+   :width: 400px
+
+A group has three components:
+
+- A list of members (accounts, representing individual people)
+- At least one set of group egress keys
+- A list of servers (or more precisely IPs)
+
+Servers list
+************
+
+The servers list is actually a list of IPs, or IP blocks. They map to your servers, network devices, or anything else with SSH capability that has an IP (on which the egress group key has been installed). Technically, this list is actually composed of 3-tuple items: remote user, remote IP (or IP block), remote port. That which applies to the personal accesses, also applies here: adding a server to the list doesn't magically give access to it, it is first necessary to install the **egress group public key**. Of course, managing the installation of these keys manually quickly becomes impractical, but you can consider these part of the configuration of the servers, hence they should be managed with whichever centralized configuration system you already use (Puppet, Chef, Ansible, /bin/cp… wait, no, strike this last one).
+
+Members list
+************
+
+The members are people who can connect to any server listed in the group server list. They'll be using the **private egress group key** they have access to, as members of said group. Of course, they have no way to extract this private key for their own use outside of the bastion, they can only use it through the bastion's code logic.
+
+Got a new team member? Just add them as a member of your group, and they instantly get access to all the group servers. Somebody leaves the company? Just delete there account on the bastion, and all the accesses are instantly gone. This is the case because all your servers should have incoming SSH sessions limited to your bastions. This way, any rogue SSH key that would have been added, is no longer of any use.
```

Comment:
```suggestion
Got a new team member? Just add them as a member of your group, and they instantly get access to all the group servers. Somebody leaves the company? Just delete their account on the bastion, and all the accesses are instantly gone. This is the case because all your servers should have incoming SSH sessions limited to your bastions. This way, any rogue SSH key that would have been added, is no longer of any use.
```